src/Service.Tests/Unittests/ConfigValidationUnitTests.cs

```diff
@@ -1525,31 +1524,25 @@ string[] linkingTargetFields
                 DataApiBuilderException ex;
                 if (apiType is ApiType.REST)
                 {
-                    ex = Assert.ThrowsException<DataApiBuilderException>(() =>
-                    RuntimeConfigValidator.ValidateRestPathForRelationalDbs(configuration));
-
                     if (pathContainsReservedCharacters)
                     {
-                        expectedErrorMessage = RuntimeConfigValidator.INVALID_REST_PATH_WITH_RESERVED_CHAR_ERR_MSG;
-                    }
-                    else
-                    {
-                        expectedErrorMessage = "REST " + expectedErrorMessage;
+                        expectedErrorMessage =
```

Comment:
Why the recalculation of the `expectedErrorMessage` if a value is provided in the DataRow? Would an assert on whether the provided expected error message work instead?

Reply:
Again, not a change from this PR. But a value was not provided for every datarow in the test- specifically for the ones that contain reserved characters. And that's why the calculation. I will update so that the test does not have to deal with this. Thanks!